cmd/tether/main_darwin.go

```diff
@@ -26,7 +27,12 @@ import (
 )
 
 func main() {
-	defer halt()
+	defer func() {
```

Comment:
Oh I see what you are saying. Yeah not sure if it catches a panic from a function that's called from main